src/lib/swapper/swappers/utils/constants.ts

```diff
@@ -1,3 +1,4 @@
 export const APPROVAL_GAS_LIMIT = '100000' // Most approvals are around 40k, we've seen 72k in the wild, so 100000 for safety.
 export const DEFAULT_SLIPPAGE = '0.002' // .2%
 export const ALLOWABLE_MARKET_MOVEMENT = '0.01' // 1%
+export const MAX_ALLOWANCE = '100000000000000000000000000'
```

Comment:
Can also remove from THOR, ZRX and CoW utils and consume the common one while at it

Reply:
https://github.com/shapeshift/web/pull/4396/commits/d58b8db4f8e7271b294bbbf3e02c13465b0369f9